src/main/java/com/terraformersmc/terrestria/init/TerrestriaSurfaces.java

```diff
@@ -82,6 +91,22 @@ public static void init() {
 			Blocks.SANDSTONE.getDefaultState(),
 			Blocks.SAND.getDefaultState()
 		);
+		
+		RANDOM_BUILDER = register("random", new RandomSurfaceBuilder());
+		
+		OUTBACK_CONFIG = new RandomSurfaceConfig(
```

Comment:
Surface configs should be serializable data, not arbitrary code

Reply:
imagine writing good code